lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -923,7 +1011,15 @@ def get_edgecolor(self):
         return self._maybe_depth_shade_and_sort_colors(super().get_edgecolor())
 
 
-def patch_collection_2d_to_3d(col, zs=0, zdir='z', depthshade=True, axlim_clip=False):
+def patch_collection_2d_to_3d(
+    col,
+    zs=0,
+    zdir="z",
+    depthshade=None,
+    axlim_clip=False,
+    *args,
```

Comment:
You aren't using `args`
```suggestion
    *,
```